### ISSUE TITLE
New issue while using the training notebook in colab

### ISSUE THREAD

**Issue by kingafy** [JohnGiorgi/DeCLUTR#218]
While executing the below script of Training notebook I am getting the below error:-
!allennlp train "declutr_small.jsonnet" \
    --serialization-dir "output" \
    --overrides "$overrides" \
    --include-package "declutr" \
    -f

```bash
2021-05-13 12:00:06.618999: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
2021-05-13 12:00:08,293 - INFO - transformers.file_utils - TensorFlow version 2.4.1 available.
Traceback (most recent call last):
  File "/usr/local/bin/allennlp", line 8, in <module>
    sys.exit(run())
  File "/usr/local/lib/python3.7/dist-packages/allennlp/__main__.py", line 34, in run
    main(prog="allennlp")
  File "/usr/local/lib/python3.7/dist-packages/allennlp/commands/__init__.py", line 92, in main
    args.func(args)
  File "/usr/local/lib/python3.7/dist-packages/allennlp/commands/train.py", line 118, in train_model_from_args
    file_friendly_logging=args.file_friendly_logging,
  File "/usr/local/lib/python3.7/dist-packages/allennlp/commands/train.py", line 168, in train_model_from_file
    params = Params.from_file(parameter_filename, overrides)
  File "/usr/local/lib/python3.7/dist-packages/allennlp/common/params.py", line 493, in from_file
    overrides_dict = parse_overrides(params_overrides)
  File "/usr/local/lib/python3.7/dist-packages/allennlp/common/params.py", line 176, in parse_overrides
    return unflatten(json.loads(evaluate_snippet("", serialized_overrides, ext_vars=ext_vars)))
RuntimeError: STATIC ERROR: 1:111: did not expect: "}"
```

**Comment by JohnGiorgi** [comment]
The error trace seems to be complaining about `overrides`. Can you please copy/paste the statement where you set the `overrides` variable?

**Comment by kingafy** [comment]
Overrides are the same as mentioned in the training notebook.

**Comment by JohnGiorgi** [comment]
Fixed in #219, just delete the last `}` in `overrides`.

**Comment by kingafy** [comment]
Thanks a lot for your help.It worked.
